src/bika/extras/browser/listingview/bika_analysisprofiles.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from zope.component import adapts
+from zope.interface import implements
+from zope.i18n.locales import locales
+
+from bika.lims import api
+from bika.extras import is_installed
+from bika.extras.config import _
+from senaite.app.listing.interfaces import IListingView
+from senaite.app.listing.interfaces import IListingViewAdapter
+
+
+class AnalysisProfilesListingViewAdapter(object):
+    adapts(IListingView)
+    implements(IListingViewAdapter)
+
+    def __init__(self, listing, context):
+        self.listing = listing
+        self.context = context
+        self.currency_symbol = self.get_currency_symbol()
+        self.decimal_mark = self.get_decimal_mark()
+
+    def before_render(self):
+        if not is_installed():
+            return
+        self.listing.contentFilter["path"] = \
+            {"query": api.get_path(self.context),
+             "depth": 1,
+             }
+        if not self.context.bika_setup.getShowPrices():
+            for i in range(len(self.listing.review_states)):
+                if "Price" in self.listing.review_states[i]["columns"]:
+                    self.listing.review_states[i]["columns"].remove("Price")
+            if "Price" in self.listing.columns:
+                del self.listing.columns['Price']
+            return
+
+        price = [
```

Comment:
'Price' already exists in the listings column, so what you're doing here is essentially updating the toggle value from True to False. Just an insight,

Reply:
No it doesn't, it an override [from](https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/controlpanel/bika_analysisprofiles.py) and also depends on `self.context.bika_setup.getShowPrices()`

Reply:
Setup > Accounting tab
![image](https://github.com/bikalims/bika.extras/assets/10023210/c0ee6f6c-9dd7-451e-8841-a2272df479f1)